tac-talk-guidelines.md

```diff
@@ -0,0 +1,75 @@
+# Guidelines for Technical Advisory Council (TAC) talks
+
+The Technical Advisory Council (TAC) is part of the Confidential Computing Consortium (CCC), a non-profit organization under the Linux Foundation. The TAC provides a platform for discussing hardware-based trusted execution technologies, their benefits, drawbacks, and implementation approaches. We are a friendly group of a mix of industry representatives, academics, and open-source enthusiasts. The technical background of the audience members is thus a wild mix. Our TAC meetings are attended by: open-source developers that are interested in confidential computing; implementors on any level of the stack that work in industry; academics that write scientific papers about confidential computing on the software or hardwar side; cloud architects that develop solutions for cloud service providers or other large industry players; hardware architects at silicon companies that work on confidential computing features for next generation's products.
+
+Due to this mix, where the only real common ground is an interest in confidential computing, technical talks can be very interesting (and slightly unpredictable) to both the presenter and the audience. When presenting, you need to keep in mind that what you see as obvious in your work might not be known to all members in the audience. That being said, we do appreciate any talk and often enough, the Q&A after the talk ends in interesting debates.
+
+### TL;DR
+- No product pitches, just technical discussions on public topics
+- 20-30 minutes + following Q&A
+- Meetings are recorded and must follow Linux Foundation antitrust policies
+- [LFX meeting calendar](https://confidentialcomputing.io/get-involved/calendar/) for meeting invite
+- An email link to suggest a talk is at the bottom
+
+## Meeting Format
+- Technical presentations are typically 20-30 minutes
+- Followed by Q&A session (duration varies based on audience engagement)
+- Total presentation slot usually does not exceed 45 minutes
+- All meetings are public and open for anyone to participate
+- The recording of the meeting will later be published on [youtube](https://www.youtube.com/@confidentialcomputingconso5871/videos).
+- Meetings are held biweekly on Thursdays at 7am US Pacific Time (4pm Central European Time)
+- Note: Time differences may vary during daylight savings transitions
+- You can find the meeting invite and time in your time zone [on the LFX meeting calendar](https://confidentialcomputing.io/get-involved/calendar/)
```

Comment:
Consider referring to just the LFX calendar and/or the times in the governance repo. Should we ever shift the meeting time or calendar URL we don't want too many places to fix it.
https://github.com/confidential-computing/governance/tree/main/TAC/Meetings#meeting-details

Reply:
```suggestion
- You can find the meeting invite and time in your time zone [on the LFX meeting calendar](https://confidentialcomputing.io/get-involved/calendar/). For more information on the TAC meetings refer to [meeting details](https://github.com/confidential-computing/governance/tree/main/TAC/Meetings#meeting-details)
```